### PR TITLE
Channel cleaning + Adaptive

### PR DESCRIPTION
**Pull request by BellezaEmporium**
- Kept the adaptive streams only
- Deleted non working streams if I know they're not working
- Give alternatives to some links

= Strongly reduces the number of dupes on lots of playlists, and keeps them full of live channels.

### CODE REVIEW

File: channels/ir.m3u [hunk 1]
```diff
@@ -22,7 +22,7 @@ https://sdw.telewebion.com/live/aftab/playlist.m3u8
 #EXTINF:-1 tvg-id="AlAlam.ir" tvg-country="IR" tvg-language="Arabic" tvg-logo="https://i.imgur.com/jY1u1zz.png" group-title="News",Al Alam (576p)
 https://live2.alalam.ir/alalam.m3u8
 #EXTINF:-1 tvg-id="AlKawtharTV.ir" tvg-country="IR" tvg-language="" tvg-logo="" group-title="",Al Kawthar TV (360p) [Not 24/7]
```
Comment (tilfizyawn2) [inline channels/ir.m3u:24]:
```suggestion
#EXTINF:-1 tvg-id="AlKawtharTV.ir" tvg-country="IR" tvg-language="Arabic" tvg-logo="" group-title="Religious",Al Kawthar TV (360p) [Not 24/7]
```

**Review by tilfizyawn2 — CHANGES_REQUESTED** [review]

File: channels/cz.m3u [hunk 2]
```diff
@@ -1,82 +1,48 @@
...
-#EXTINF:-1 tvg-id="OckoExpres.cz" tvg-country="CZ" tvg-language="Czech" tvg-logo="" group-title="Music",Óčko Expres (540p)
-https://ocko-live.ssl.cdn.cra.cz/channels/ocko_expres/playlist/cze/live_hq.m3u8
-#EXTINF:-1 tvg-id="OCKOEXPRESmusic.cz" tvg-country="CZ" tvg-language="" tvg-logo="" group-title="Music",OCKO EXPRES Music (360p)
-http://ocko-live.ssl.cdn.cra.cz/channels/ocko_expres/playlist/cze/live_lq.m3u8?offsetSeconds=0
-#EXTINF:-1 tvg-id="OckoGold.cz" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://i.imgur.com/nrWNyLc.jpg" group-title="Music",Óčko Gold (540p)
-https://ocko-live-dash.ssl.cdn.cra.cz/cra_live2/ocko_gold.stream.1.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="OckoStar.cz" tvg-country="CZ" tvg-language="Czech" tvg-logo="" group-title="Music",Óčko Star (540p)
-https://ocko-live.ssl.cdn.cra.cz/channels/ocko_gold/playlist/cze/live_hq.m3u8
-#EXTINF:-1 tvg-id="OckoTV.cz" tvg-country="CZ" tvg-language="" tvg-logo="https://i.imgur.com/jKPRYdM.png" group-title="Music",Ocko TV (540p)
-https://ocko-live-dash.ssl.cdn.cra.cz/cra_live2/ocko.stream.1.smil/playlist.m3u8
```
Comment (Dum4G) [inline channels/cz.m3u:35]: image

All Ocko Links do work for me. Are they duplicating something?

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/fi.m3u [hunk 3]
```diff
@@ -1,21 +1,7 @@
...
-#EXTINF:-1 tvg-id="YLETV1.fi" tvg-country="INT" tvg-language="Finnish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6e/Yle_TV1_logo.svg/382px-Yle_TV1_logo.svg.png" group-title="",YLE TV 1 (720p) [Not 24/7]
-https://yletvhdliveworld-lh.akamaihd.net/i/yletv1hdworld_1@187592/master.m3u8
-#EXTINF:-1 tvg-id="YLETV2.fi" tvg-country="FI" tvg-language="Finnish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6a/Yle_TV2_logo.svg/399px-Yle_TV2_logo.svg.png" group-title="",YLE TV 2 (720p) [Geo-blocked]
-https://yletv-lh.akamaihd.net/i/yletv2hls_1@103189/master.m3u8
-#EXTINF:-1 tvg-id="YLETV2.fi" tvg-country="INT" tvg-language="Finnish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6a/Yle_TV2_logo.svg/399px-Yle_TV2_logo.svg.png" group-title="",YLE TV 2 (720p) [Not 24/7]
-https://yletvhdliveworld-lh.akamaihd.net/i/yletv2hdworld_1@187593/master.m3u8
-#EXTINF:-1 tvg-id="NebesaTV7.fi" tvg-country="RU" tvg-language="Russian" tvg-logo="https://www.nebesatv7.com/wp-content/themes/tv7-theme/assets/img/logo_nebesa_short.png" group-title="Religious",Небеса ТВ7 (720p)
-https://vod.tv7.fi/tv7-ru/tv7-ru.smil/playlist.m3u8
```
Comment (Dum4G) [inline channels/fi.m3u:21]: image
works

Comment (Dum4G) [reply channels/fi.m3u:21]: Are you sure all of those yletv links are mot worth testing?

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/fi.m3u (hunk 3 above)
Comment (LaneSh4d0w) [reply channels/fi.m3u:21]: Might have done a mistake when merging conflicts. I'm gonna take a look.

File: channels/cz.m3u [hunk 4]
```diff
@@ -1,82 +1,48 @@
...
-#EXTINF:-1 tvg-id="NovaGold.cz" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/7/7f/Nova_Gold_logo.png" group-title="",Nova Gold (640p) [Offline]
-https://nova-live.ssl.cdn.cra.cz/channels/nova_gold_avod/playlist/cze/live_xq.m3u8
-#EXTINF:-1 tvg-id="NovaGold.cz" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://static.epg.best/cz/NovaGold.cz.png" group-title="",Nova Gold (576p) [Offline]
-https://nova-live.ssl.cdn.cra.cz/channels/nova_gold_avod/playlist/cze/live_hq.m3u8
-#EXTINF:-1 tvg-id="Ocko.cz" tvg-country="CZ;SK" tvg-language="Czech" tvg-logo="" group-title="Music",Óčko (540p)
-https://ocko-live.ssl.cdn.cra.cz/channels/ocko/playlist.m3u8
-#EXTINF:-1 tvg-id="Ocko.cz" tvg-country="CZ;SK" tvg-language="Czech" tvg-logo="" group-title="Music",Óčko (540p)
-https://ocko-live.ssl.cdn.cra.cz/channels/ocko/playlist/cze/live_hq.m3u8
-#EXTINF:-1 tvg-id="OckoExpres.cz" tvg-country="CZ" tvg-language="Czech" tvg-logo="" group-title="Music",Óčko Expres (540p)
-https://ocko-live.ssl.cdn.cra.cz/channels/ocko_expres/playlist/cze/live_hq.m3u8
```
Comment (Dum4G) [inline channels/cz.m3u:27]: Looks like a missed channel

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/cz.m3u (hunk 4 above)
Comment (LaneSh4d0w) [reply channels/cz.m3u:27]: fixed

**Review by Dum4G — APPROVED** [review]

**Pull request merged**